src/EnergyPlus/HeatBalanceIntRadExchange.cc

```diff
@@ -1710,6 +1747,19 @@ namespace HeatBalanceIntRadExchange {
                     state, "FixViewFactors: View factors not complete. Check for bad surface descriptions or unenclosed zone=\"" + enclName + "\".");
             }
         }
+        if (severeErrorPresent) {
+            ShowFatalError(state,
+                           "FixViewFactors: View factor calculations significantly out "
+                           "of tolerance.  See above messages for more information.");
+        }
+    }
+
+    bool DoesZoneHaveInternalMass(EnergyPlusData &state, int const numZoneSurfaces, const Array1D_int &surfPointer)
```

Comment:
`state` is unused here.
Also, minor detail, but `numZoneSurfaces` isn't necessary. The loop statement could be ` for (int surfNum : surfPointer)`.
If I'm following correctly, this only happens once per enclosure? So, there's no concern about repeating this, correct?

Reply:
Yes, should happen once (at most twice--not exactly sure why it's coming through twice sometimes?) per enclosure.  So, basically, get rid of state and numZoneSurfaces and adjust loop to the suggestion.  Will do.

Reply:
You sure I don't need state?  It's used in the line:

if (**_state_**.dataSurface->Surface(surfPointer(surfNum)).Class == SurfaceClass::IntMass)...

My compiler (after getting rid of state) is unhappy and I've tried a variety of variations.  Of course, maybe I just don't understand "state"...

Reply:
Oh, sorry, missed that.

Reply:
The `for (int surfNum : surfPointer)` didn't work...this happens once per zone.  The second time it comes into this routine (for the second zone), the counter picks up where it left off and then I get a crash.  In the file that I was running, there were 9 surfaces in the first zone and six in the second.  The first time through there weren't any problems.  The second time through for zone 2 the counter starts at 10 which is more than the number of surfaces in the zone and I get a pointer issue.  I think I need to go back to what I originally had?

Reply:
It's fine as-is, but for future reference, there's no counter with the suggested form, so this should work fine for multiple calls.
```
for (int surfNum : surfPointer){
 if (state.dataSurface->Surface(surfNum).Class == SurfaceClass::IntMass) return true;
}
```